_data/events/gdcr22-krakow

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Needs to end in .json so we process it 😊

Reply:
✅ Sorry, my bad!